assets/src/admin/site-scan-notice/index.js

```diff
@@ -67,10 +71,18 @@ Providers.propTypes = {
 };
 
 domReady( () => {
-	const root = document.getElementById( APP_ROOT_ID );
+	let root = document.getElementById( APP_ROOT_ID );
 
 	if ( ! root ) {
-		return;
+		const rootSibling = document.getElementById( APP_ROOT_SIBLING_ID );
+
+		if ( ! rootSibling ) {
+			return;
+		}
+
+		root = document.createElement( 'div' );
+		root.setAttribute( 'id', APP_ROOT_ID );
+		rootSibling.after( root );
```

Comment:
Nice. The `after` method is available for every browser except IE11. I need to start using it!